smart_open/smart_open_lib.py

```diff
@@ -635,12 +635,26 @@ def compression_wrapper(file_obj, filename, mode):
         return file_obj
 
 
-def file_smart_open(fname, mode='rb'):
+def file_smart_open(fname, mode='rb', parents=False):
     """
     Stream from/to local filesystem, transparently (de)compressing gzip and bz2
     files if necessary.
 
     """
+    if parents:
+        from ntpath import split
```

Comment:
Why you use `ntpath` instead of `os.path` that use more suitable `path module` for your OS?
Look to [python os.path doc](https://docs.python.org/2/library/os.path.html)